dask_expr/_reductions.py

```diff
@@ -967,7 +1000,7 @@ def aggregate(cls, results: list[pd.Series], dropna=None):
         [df] = results
         max = df.max(skipna=dropna)
         out = df[df == max].index.to_series().sort_values().reset_index(drop=True)
-        out.name = results[0].name
+        # out.name = results[0].name
```

Comment:
Are we removing this line?

Reply:
Oh yes, frorgot to delete. I don't know why tests were passing before...